.changeset/famous-numbers-happen/changes.json

```diff
@@ -0,0 +1 @@
+{ "releases": [{ "name": "@keystone-alpha/app-graphql", "type": "patch" }], "dependents": [] }
```

Comment:
I don't think this file should exist with the latest changesets. Make sure your branch is from the HEAD of `master` and that you've reinstalled all your dependencies before rerunning the `changeset` command.

Reply:
Changesets 2 can handle changesets in this format so it's totally fine to leave this btw.